src/content/release/breaking-changes/image-filter-blur-tilemode.md

```diff
@@ -0,0 +1,82 @@
+---
+title: ImageFilter.blur default tile mode automatic selection.
+description: >-
+  If a tile mode wasn't specified in the constructor, ImageFilter.blur will
+  select one based on the rendering context.
+---
+
+## Summary
+
+  The `ui.ImageFilter.blur`'s default tile mode is now automatically selected
+  by the backend. Previously `TileMode.clamp` was used unless a different tile
+  mode was specified. Passing a value of `null` to the filter's constructor
+  specifies automatic selection.
```

Comment:
Does this imply that you need to manually pass a null to get the behavior? How about:

"Previously the default was `clamp` unless a different tile mode was specified. Now, the default is `null` and specifies automatic selection unless a specific tile mode is specified."